tensorflow_lite_support/examples/task/text/desktop/python/nl_classifier_demo.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+import argparse
+import os
+
+def classify(model_path, text):
+    """Classifies input text into different categories
+
+    Initially the function will download the model and then runs the detection tool.
```

Comment:
Will this function download models?

Reply:
No it doesnt. Previously I added the download option to this function and later I removed it. Sorry I forgot to remove it from the docstring. Removed it in the latest commit